python/pyspark/sql/tests.py

```diff
@@ -3966,6 +3967,15 @@ def random_udf(v):
         random_udf = random_udf.asNondeterministic()
         return random_udf
 
+    def test_pandas_udf_tokenize(self):
+        from pyspark.sql.functions import pandas_udf
+        tokenize = pandas_udf(lambda s: s.apply(lambda str: str.split(' ')),
```

Comment:
instead of s.apply with a lambda, you could do `lambda s: s.str.split(' ')`, both do the same just a little more compact

Reply:
hm. I thought this PR targets to clarify array type wuth primitive types. can we improve the test case here -https://github.com/holdenk/spark/blob/342d2228a5c68fd2c07bd8c1b518da6135ce1bf6/python/pyspark/sql/tests.py#L3998, and remove this test case?

Reply:
I prefer to have a test case for primitive  array type as well.

Reply:
I think this is a pretty common use to tokenize, so I think it's fine to have an explicit test for this

Reply:
I dont think this PR targets to fix or support tokenizing in an udf ..

Reply:
@HyukjinKwon It doesn't, but given that the old documentation implied that the ionization usecase wouldn't work I thought it would be good to illustrate that it does in a test.